metaflow/plugins/kfp/constants.py

```diff
@@ -3,9 +3,11 @@
 # Defaults for running MF on KFP
 # TODO: remove this when we're able to automatically package code and use it on KFP. We currently need the user supplied code URL (or this default URL) in order to download the code that is to be run on KFP
 DEFAULT_FLOW_CODE_URL = 'https://gist.githubusercontent.com/mon95/98d9d88571a0307a53b637d841295702/raw/df8bc1fa363f629271cfc1d34f3c2f7ca0b2e2cf/helloworld.py'
+DEFAULT_DOWNLOADED_FLOW_FILENAME = 'downloaded_flow.py'
```

Comment:
If this is not used anywhere, then let's delete.  thanks

Reply:
Using this in building the command templates (outside of the container ops)